### ISSUE TITLE
hover state always affect the result

### ISSUE THREAD

**Issue by nanjixiong218** [gemini-testing/gemini#707]
sometimes the same test code get diff result picture because of the hover state；
so the expected png and the actual png will be different when run times;
just like:

![image](https://cloud.githubusercontent.com/assets/5085933/21536194/623180f8-cdba-11e6-9559-2efd6843c766.png)

sometimes it passed,sometimes it failed;

i have done some time waiting;

![image](https://cloud.githubusercontent.com/assets/5085933/21536212/ad203e74-cdba-11e6-9c72-3e784419838d.png)

**Comment by nanjixiong218** [comment]
@sipayRT can you have a look?

**Comment by HeeL** [comment]
I have the same problem. The code that I didn't change for a month, stopped working in the way that tests are failing because of the hover state.

It seems like the hover state happens  only by `update` task.

**Comment by HeeL** [comment]
btw, quick hack fix for us was:

```js

            .setUrl('/hi/no-images')
            .before(function (actions) {
                actions.focus('body');
            })
            .capture('plain');


```

**Comment by gilligan** [comment]
has anyone looked into this at all? Quite an issue for us ;(

**Comment by HeeL** [comment]
Correct me if I'm wrong, but the problem is in the dependency of gemini. Can some core-team comment on this and mark as won't-fix if needed?

**Comment by gilligan** [comment]
@HeeL uhrm.. if the problem with the dependency basically breaks gemini core functionality I don't think that setting this problem to `wont-fix` would be appropriate

**Comment by HeeL** [comment]
Yes, actually the version of this package could be downgraded till the problem will be solved in newer versions.

**Comment by DudaGod** [comment]
Hello guys.
I apologize that we took so long to answer. This problem is still reproduced?
Can you show me your gemini config and specify please the used gemini version?

In our projects I haven't met such a problem...

**Comment by HeeL** [comment]
@DudaGod I guess one of the reason why you don't hit this problem is because you are not using image composition for creating visual tests for long scrollable pages. By doing selective captures, you don't experience this problem.

**Comment by artemyarulin** [comment]
Any progress on this? Just found the same and weird thing it's not consistent for some reason. 

I remember reading in the docs that before suite Gemini will reset mouse position to avoid any side affects, but for doing capture it will point mouse into captured element center.
